### PR TITLE
MULTIARCH-4095: Generate PowerVS machine manifests for CAPI

### PR DESCRIPTION
**Pull request by hamzy**
Generate the PowerVS master and bootstrap machine manifests for CAPI.

### CODE REVIEW

**Comment by hamzy** [comment]
/assign @patrickdillon 
I don't know if this is the final version yet, but I would like to commit these changes in order to avoid having to repeatedly fix merge conflicts in the vendoring.

File: pkg/infrastructure/powervs/clusterapi/powervs.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ type Provider struct {
 }
 
 var _ clusterapi.PreProvider = Provider{}
+var _ clusterapi.Provider = Provider{}
```
Comment (patrickdillon) [inline pkg/infrastructure/powervs/clusterapi/powervs.go]: nit: more idiomatic to do:

```go
var _ clusterapi.PreProvider = (*Provider)(nil)
var _ clusterapi.Provider = (*Provider)(nil)
```

**Review by patrickdillon — APPROVED** [review]

File: pkg/infrastructure/powervs/clusterapi/powervs.go (hunk 1 above)
Comment (hamzy) [reply pkg/infrastructure/powervs/clusterapi/powervs.go]: Changed, thanks!

**Pull request merged**